wiki/Tournaments/CWC/2019/en.md

```diff
@@ -0,0 +1,243 @@
+---
+tags:
+- CWC 2019
+- CWC2019
+---
+osu!catch World Cup 2019
+=============================
+
+![osu!catch World Cup 2019](img/logo.png)
```

Comment:
This image too big, probably better to use the actual banner like on [this article](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019) for example